components/txn_types/src/lock.rs

```diff
@@ -67,13 +68,15 @@ pub struct Lock {
     pub for_update_ts: TimeStamp,
     pub txn_size: u64,
     pub min_commit_ts: TimeStamp,
+    pub is_parallel_commit: bool,
```

Comment:
Should we use the same name "async commit" as in `kvproto`? Maybe we also need to change the documents.

Reply:
I've renamed it here.